src/api/app/models/review.rb

```diff
@@ -230,6 +231,26 @@ def set_reviewable_association
   def validate_reviewer_fields
     (by_user && (by_group || by_project || by_package)) || (by_group && (by_project || by_package))
   end
+
+  def update_cache
+    if user_id
+      user_ids = [user_id]
+    elsif group_id
+      user_ids = GroupsUser.where(group_id: group_id).pluck(:user_id)
+    elsif package_id
+      user_ids = Relationship.joins(:groups_users).where(package_id: package_id).pluck('groups_users.user_id')
+      user_ids += Relationship.where(package_id: package_id).where.not(user_id: nil).pluck(:user_id)
+    elsif project_id
+      user_ids = Relationship.joins(:groups_users).where(project_id: project_id).pluck('groups_users.user_id')
+      user_ids += Relationship.where(project_id: project_id).where.not(user_id: nil).pluck(:user_id)
+    end
+
+    # rubocop:disable Rails/SkipsModelValidations
+    # Skipping Model validations in this case is fine as we only want to touch
+    # the associated user models to invalidate the cache keys
+    User.where(id: user_ids).update_all(updated_at: Time.now)
```

Comment:
Don't really understand how you can access `user_ids`. Is it an attribute oder `Review` ?